cli/test_util.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+use std::path::PathBuf;
+use std::process::Child;
+use std::process::Command;
+use std::sync::Mutex;
+use std::sync::MutexGuard;
+use std::thread;
+use std::time;
+
+lazy_static! {
+  static ref GUARD: Mutex<()> = Mutex::new(());
+}
+
+// Warning: broken from other crates?
+pub fn root_path() -> PathBuf {
+  PathBuf::from(concat!(env!("CARGO_MANIFEST_DIR"), "/.."))
+}
+
+pub fn deno_exe_path() -> PathBuf {
+  let root = root_path();
+  let mut target = "debug";
+  if let Ok(build_mode) = std::env::var("DENO_BUILD_MODE") {
```

Comment:
I think we don't set DENO_BUILD_MODE anymore in travis.

Something like the below worked on my machine for switching debug and release.
```rust
#[cfg(not(debug_assertions))]
let target = "release";
#[cfg(debug_assertions)]
let target = "debug";
```

Reply:
I did something else using current_exe... not sure it's more stable yet.